src/Transform/ONNX/ReplaceNoValueOperands.cpp

```diff
@@ -117,12 +117,43 @@ class Conv2DReplaceNoValue : public OpRewritePattern<ONNXConvOp> {
   }
 };
 
+class ResizeReplaceNoValue : public OpRewritePattern<ONNXResizeOp> {
```

Comment:
`test/mlir/onnx/onnx_replace_novalue.mlir` should be extended to test this pattern.